great_expectations/rule_based_profiler/parameter_builder/parameter_builder.py

```diff
@@ -179,6 +180,7 @@ def build_parameters(
             batch_list: Explicit list of Batch objects to supply data at runtime.
             batch_request: Explicit batch_request used to supply data at runtime.
             force_batch_data: Whether or not to overwrite existing batch_request value in ParameterBuilder components.
+            ignore_existing_parameters: If "True", recompute parameter value if "fully_qualified_parameter_name" exists.
```

Comment:
Can you explain why this is needed?

Reply:
So my understanding is that this allows us to re-use and overwrite an existing parameter. Is that correct?

Reply:
Should this be something like `overwrite_existing_parameter` or `reuse_existing_parameter`? `ignore` sounds like the value will persist, but just be ignored.